Exercise_D and E/index.html

```diff
@@ -9,14 +9,41 @@
     <link rel="stylesheet" href="./assets/style/index.css" />
     <link
       rel="stylesheet"
-      href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+      href=""
```

Comment:
interesting that you removed bootstrap!


You can achieve your bottom section using the bootstrap column system to ensure that it is responsive. 

But I like your grid implementation